### PR TITLE
Crèpes recipe added

### PR DESCRIPTION
**Pull request by ZeikoFr**
Hi, added a classic french recipe for breakfast and dessert. Hope you like it !

### CODE REVIEW

**Comment by m-chrzan** [comment]
There already is a [French crepes](https://based.cooking/french-crepes) recipe. Does this one differ significantly?

File: src/crepes.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Crèpes
+
+
+- ⏲️ Prep time: 10 min
+- 💤 Rest time: up to 24 hours for a better taste but can be done immediatly after prep
```
Comment (hallzy) [inline src/crepes.md:5]:
```suggestion
- 💤 Rest time: up to 24 hours for a better taste but can be done immediately after prep
```

File: src/crepes.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- 🍽️ Servings: 2
+
+## Ingredients
+
+- 0.5L of milk (cow milk, almond milk, the one you prefer)
+- 2 eggs
+- 200g flour
+- 1 bag of vanilla sugar
+- 1 teaspoon of oil (preferably sunflower oil)
+- a bit of salt
```
Comment (hallzy) [inline src/crepes.md:16]:
```suggestion
```

No need for salt/pepper in ingredients list per the contribution instructions

File: src/crepes.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+1. Put the milk in your recipient/blender.
+2. Put 100g of flour while blending.
+3. Once the first 100g are well mixed add the other 100g.
+4. Add the eggs.
+5. Add the vanilla sugar, oil and salt.
+6. Blend everything until you get a good homogeneous batter with some little bubble on top.
+7. Heat up a pan really hot and put a little bit of butter in it and spread it everywhere.
+8. Take an estimated amount of crepe batter that will cover the entire pan and spread it on the pan.
+9. After 45 seconds try to turn the crepe over, it should not stick to the pan.
+10. Once the other side id cooked put the crepe on a plate and start another one (adjust the quantity of batter if necessary.
```
Comment (hallzy) [inline src/crepes.md:30]:
```suggestion
10. Once the other side is cooked put the crepe on a plate and start another one (adjust the quantity of batter if necessary.
```

File: src/crepes.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+11. After 4/5 crepes you might need to re-add a bit of butter in the pan.
+
+## Contribution
+
+
+;tags: dessert breakfast french
```
Comment (hallzy) [inline src/crepes.md:36]:
```suggestion

;tags: dessert breakfast french
```

No need for the contribution heading if you aren't putting your name in.

File: src/crepes.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# Crèpes
```
Comment (hallzy) [inline src/crepes.md:1]:
```suggestion
# Crêpes
```

**Review by hallzy — CHANGES_REQUESTED** [review]
See suggested changes.

Also, as pointed out by another user, there is a Crêpe recipe already, so unless there is a significant difference, I don't know if this will be merged.

Maybe a better option would be to update the original if you think it is lacking in some way.

**Comment by ZeikoFr** [comment]
Yeah I didn't see the first recipe I close the PR and will update the original one

**Pull request closed without merging**